builtin/v13/market/market_types.go

```diff
@@ -138,3 +139,31 @@ type GetDealActivationReturn struct {
 type GetDealSectorParams = DealQueryParams
 
 type GetDealSectorReturn = abi.SectorNumber
+
+type SettleDealPaymentsParams struct {
+	Deals bitfield.BitField
+}
+
+type SettleDealPaymentsReturnResultFailCodes struct {
```

Comment:
You can just call this a `FailCode`.

Reply:
(we may want to move it to a shared location later anyways)

Reply:
Yeah, we'll move both `FailCode` and `BatchReturn` to `util` in a followup patch (removing it from the verifreg.